src/Common/FrequencyHolder.h

```diff
@@ -0,0 +1,237 @@
+#pragma once
+#include <Common/StringUtils/StringUtils.h>
+#include <IO/ReadBufferFromFile.h>
+#include <IO/ReadBufferFromString.h>
+#include <IO/ReadHelpers.h>
+#include <IO/readFloatText.h>
+#include <IO/Operators.h>
+#include <IO/ZstdInflatingReadBuffer.h>
+
+#include <Common/Arena.h>
+#include <base/StringRef.h>
+#include <Common/HashTable/HashMap.h>
+
+#include <string_view>
+#include <string>
+#include <cstring>
+#include <unordered_map>
+#include <base/logger_useful.h>
+#include <Common/getResource.h>
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int FILE_DOESNT_EXIST;
+}
+
+class FrequencyHolder
```

Comment:
Please add a comment here, like that this class is needed for loading text classification data ...